manifests/crd.yaml

```diff
@@ -6,8 +6,47 @@ metadata:
 spec:
   group: helm.cattle.io
   version: v1
+  additionalPrinterColumns:
```

Comment:
This is "safe" and has no backwards compatibility impacts?

Reply:
You can add and remove these ad-hoc at runtime. Kubectl and server-side printing uses the entries in this field to determine what fields to display in get and describe output. If they're missing, it just prints name and namespace.